ergoCubSN002/hardware/inertials/left_arm-eb4-j2_3-inertial.xml

```diff
@@ -15,7 +15,7 @@
             <group name="PROPERTIES">
             
                 <group name="CANBOARDS">
-                    <param name="type">                 mtb4c                </param>
+                    <param name="type">                 mtb4                </param>
```

Comment:
Just inquiring about this change.
It was introduced by @filippoborgogni via https://github.com/icub-tech-iit/robots-configuration/commit/688614c0ac1653ca881c76cac85693a34e9005bd.

However:
- This device is not launched as inertials are commented out in the main xml.
- The [network](https://github.com/icub-tech-iit/robots-configuration/blob/devel-ergoCubSN002/ergoCubSN002/network.ergoCubSN002.xml) file still points to `mtb4c`.

Reply:
@AntonioConsilvio updated the network file consistently.
Even though the inertials are not launched, we try to keep board versions up-to-date.